File: app/src/main/java/org/michaelbel/application/rest/model/Account.java
package org.michaelbel.application.rest.model;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("all")
public class Account {

    @SerializedName("avatar")
    public Avatar avatar;

    @SerializedName("id")
    public int id;

    @SerializedName("iso_639_1")
    public String iso_639_1;

    @SerializedName("iso_3166_1")
    public String iso_3166_1;

    @SerializedName("name")
    public String name;

    @SerializedName("include_adult")
    public boolean includeAdult;

    @SerializedName("username")
    public String username;

    public class Avatar {

        public class GAvatar {

            @SerializedName("hash")
            public String hash;
        }
    }
}

File: data/src/main/java/org/michaelbel/data/remote/model/Account.kt
package org.michaelbel.data.remote.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Account(
        @Expose @SerializedName("avatar") val avatar: Avatar,
        @Expose @SerializedName("id") val id: Int,
        @Expose @SerializedName("iso_639_1") val lang: String,
        @Expose @SerializedName("iso_3166_1") val country: String,
        @Expose @SerializedName("name") val name: String,
        @Expose @SerializedName("include_adult") val include_adult: Boolean,
        @Expose @SerializedName("username") val username: String
): Serializable